roles/girder-post-install/files/girder-post-install.py

```diff
@@ -0,0 +1,168 @@
+#! /usr/bin/env python
+
+import json
+import os.path
+
+from argparse import ArgumentParser
+from time import sleep
+
+from girder.constants import AssetstoreType
+from girder_client import GirderClient
+
+def rest(client, method, *args, **kwds):
+    method = getattr(client, method)
+    old_parameters = kwds.pop('parameters')
+    kwds['parameters'] = {}
+    kwds['parameters'].update(
+        (k, v[1:-1]) if (v.startswith('"') and v.endswith('"')) else (k, v)
+        for k, v in (
+            (key, json.dumps(value))
+            for key, value in old_parameters.items()))
+
+    print kwds['parameters']
+    return method(*args, **kwds)
+
+def post(client, *args, **kwds):
+    return rest(client, 'post', *args, **kwds)
+
+def put(client, *args, **kwds):
+    return rest(client, 'put', *args, **kwds)
+
+def get(client, *args, **kwds):
+    return rest(client, 'get', *args, **kwds)
```

Comment:
We actually have all these methods inside `GirderClient` already. https://github.com/girder/girder/blob/master/clients/python/girder_client/__init__.py#L228-L238